macros/helpers/nozzle_cleaning.cfg

```
@@ -120,7 +121,7 @@ gcode:
         # Retract
         G92 E0
         G1 E-1.7 F2100
-        G1 E-18.3 F150
+        G1 E-({RETRACT}-1.7) F150
```

Can you replace it by like 10% like on the following suggestion? It should be pretty equivalent but also more compatible with any value of RETRACT.

```suggestion
        G1 E-({RETRACT}-{RETRACT*0.1}) F150
```